src/main/java/frc/robot/Robot.java

```diff
@@ -86,7 +91,7 @@ public void robotInit() {
         DashboardUtils.logTelemetry();
 
         drive.setDefaultCommand(drive.drive(driveController::getTriggers, () -> driveController.getX(Hand.kLeft)));
-        lift.setDefaultCommand(lift.moveLift(copilotController::getTriggers));
+        lift.setDefaultCommand(lift.moveLift(() -> driveController.getY(Hand.kRight)));
```

Comment:
I expect that the wiki page on controls will be updated?

Reply:
Added it